cms/src/main/java/uk/nhs/digital/externalstorage/s3/S3ConnectorImpl.java

```diff
@@ -45,7 +67,21 @@ public S3ObjectMetadata uploadFile(InputStream fileStream, String fileName, Stri
         ObjectMetadata metadata = new ObjectMetadata();
         metadata.setContentType(contentType);
 
-        s3.putObject(new PutObjectRequest(bucketName, objectKey, fileStream, metadata));
+        // initialise multipart upload
+        InitiateMultipartUploadResult initResult = s3.initiateMultipartUpload(new InitiateMultipartUploadRequest(bucketName, objectKey, metadata));
+        initResult.getUploadId();
+
+        // loop parts
+        List<PartETag> partETags = new ArrayList<>();
+        try {
+            partETags = uploadParts(fileStream, bucketName, objectKey, initResult.getUploadId());
+        } catch (IOException ex) {
+            log.error("Error while uploading file", ex);
+            s3.abortMultipartUpload(new AbortMultipartUploadRequest(bucketName, objectKey, initResult.getUploadId()));
+        }
+
+        // finalise multipart upload
+        s3.completeMultipartUpload(new CompleteMultipartUploadRequest(bucketName, objectKey, initResult.getUploadId(), partETags));
```

Comment:
The unit test doesn't seem to verify that the 'complete request' call takes place.